docs/usage.rst

```diff
@@ -327,19 +327,14 @@ logging handler can use different transports. The default is
 fluentd logging handlers
 ~~~~~~~~~~~~~~~~~~~~~~~~
 
-Besides :class:`~google.cloud.logging.handlers.handlers.CloudLoggingHandler`,
-which writes directly to the API, two other handlers are provided.
-:class:`~google.cloud.logging.handlers.app_engine.AppEngineHandler`, which is
-recommended when running on the Google App Engine Flexible vanilla runtimes
-(i.e. your app.yaml contains ``runtime: python``), and
```

Comment:
We should mark the classes as deprecated: update the docs to mention that they are not returned anymore and that new classes replace them. Folks who have existing code using 2.* library release should have a path forward that explains how to upgrade to 3.0.0

Reply:
My plan was to go through and update the docs at the end of the milestone, just in case any docs changes become out of date later (https://github.com/googleapis/python-logging/issues/245)

Do you think that makes sense, or should I try to update the docs along-side changes?

Reply:
We can update the docs later. The point here is more about the annotation of the classes.